test/test_jit.py

```diff
@@ -1409,7 +1409,7 @@ def fn(x, y, z):
     def _make_scalar_vars(self, arr, dtype):
         out = []
         for inp in arr:
-            out.append(Variable(torch.from_numpy(np.array([inp], dtype=dtype))))
+            out.append(torch.from_numpy(np.array(inp, dtype=dtype)))
```

Comment:
cc @jamesr66a @zdevito  

The script seems to process these as scalars (0-d tensors) after Tensor/Variable merge